projects/igniteui-angular/src/lib/grids/column.component.ts

```diff
@@ -718,6 +718,34 @@ export class IgxColumnComponent implements AfterContentInit {
             this.grid.cdr.markForCheck();
         }
     }
+    /**
+     * Returns a reference to the `quickFilterTemplate`.
+     * ```typescript
+     * let quickFilterTemplate = this.column.quickFilterTemplate;
+     * ```
+     * @memberof IgxColumnComponent
+     */
+    @Input('quickFilterTemplate')
```

Comment:
Could you please change the name to `filterCellTemplate`. It's a bit more meaningful for the users.